PubnativeLite/PubnativeLite/Ad Model/HyBidAdSize.h

```diff
@@ -44,6 +45,7 @@
 @property (class, nonatomic, readonly) HyBidAdSize *SIZE_NATIVE;
 
 - (BOOL)isEqualTo:(HyBidAdSize *)hyBidAdSize;
+- (HyBidAdSize *)convertSizeToHyBid:(CGSize)size;
```

Comment:
maybe naming this to `convertSizeToHyBidAdSize` is more informative? :what do you think @orkhanalizada2020 ?

Reply:
I used Swifty way, when the name of the parameter completes the name of the method 😅
Changed it! :) @can-soykarafakili